src/main/java/org/prebid/server/bidder/telaria/TelariaBidder.java

```diff
@@ -148,24 +130,33 @@ private ExtImpTelaria parseImpExt(Imp imp) {
         }
     }
 
-    private Site modifySite(String seatCode, Site site) {
+    private Imp updateImp(Imp imp, ExtImpTelaria extImp, String publisherId) {
+        if (StringUtils.isBlank(extImp.getSeatCode())) {
+            throw new PreBidException("Telaria: Seat Code required");
+        }
+        return imp.toBuilder()
+                .tagid(extImp.getAdCode())
+                .ext(mapper.mapper().valueToTree(ExtImpOutTelaria.of(imp.getTagid(), publisherId)))
+                .build();
+    }
+
+    private static Site modifySite(String seatCode, Site site) {
```

Comment:
Pls flip arguments so `site` will be the first since we name the method `modifySite(..)`
Also, pls check similar in modifyApp, createPublisher, etc.